nextcloud/Dockerfile

```diff
@@ -26,7 +26,10 @@ FROM ${BUILD_FROM}
 ENV NEXTCLOUD_PATH="/data/config/www/nextcloud"
 
 RUN \
-    chmod +x /defaults/nextcloud-perms.sh \
+    # Remove upstream files
```

Comment:
avoids duplication with the files you renamed

Reply:
nice one :)